### DISCUSSION TITLE
Does it have any kind of support for offline applications?

### DISCUSSION THREAD

**Discussion by alam94h** [amplitude/Amplitude-TypeScript#210] [Q&A]
Hi!

I wish to know if there is any kind of support for offline applications.

We are working with Android and iOS using cordova and in our case our users usually use the apps offline so we where wondering if this library has the capability to keep a database of sorts in order to ensure the packages are send when the connection is established.

We found that the Amplitude-iOS library does have it, so we where wondering if this was also brought here.

**Comment by malaidev** [comment]
I see it for real, now.
